addons/website_sale/models/sale_order.py

```diff
@@ -490,6 +490,7 @@ def _action_confirm(self):
             # name, street, city, state, zip, country
             name = order.partner_shipping_id.name
             street = order_location['pick_up_point_address']
+            house_number = order_location['house_number']
```

Comment:
Hello :wave: 

It might be a good idea to keep the same name convention as for the others pick up points informations.
```suggestion
            house_number = order_location['pick_up_point_house_number']
```

It was not possible to add the house number previously because of Sendcloud's API, and that does not appear to have changed yet (cfr https://api.sendcloud.dev/docs/sendcloud-public-api/pickups/operations/get-a-pickup), the house number is directly added in the 'pick_up_point_address'.

I think that the main problem here is that we never use the "address_2" line sent by the API, which may contain the house number.

Also, I don't see an enterprise pr linked in the commit message, so I'm assuming that you correctly updated the [website_delivery_sendcloud](<https://github.com/odoo/enterprise/blob/1233fc0ae6c43f04bd2e9ecc1e3784e5c93b2971/website_delivery_sendcloud/models/delivery_sendcloud.py#L90>) module. 
And to be more consistent, this could also be applied to [Fedex](<https://github.com/odoo/enterprise/blob/1233fc0ae6c43f04bd2e9ecc1e3784e5c93b2971/website_sale_fedex/models/delivery_fedex.py#L37>) by taking all the "Streetlines" into account [here](<https://github.com/odoo/enterprise/blob/1233fc0ae6c43f04bd2e9ecc1e3784e5c93b2971/website_sale_fedex/models/delivery_fedex.py#L34>).